themes/edx.org/lms/templates/footer.html

```diff
@@ -52,22 +52,26 @@ <h2 class="sr footer-about-title">${_("About edX")}</h2>
       </div>
 
       <div class="external-links">
-        <div class="social-media-links">
-          % for link in footer['social_links']:
-          <a href="${link['url']}" class="sm-link external" title="${link['title']}" rel="noreferrer">
-            <span class="icon fa ${link['icon-class']}" aria-hidden="true"></span>
-            <span class="sr">${link['action']}</span>
-          </a>
-          % endfor
-        </div>
+        <ul class="social-media-links">
+            % for link in footer['social_links']:
+            <li>
```

Comment:
It's a list of links that were just tossed into a container. When they were being announced, they really didn't have any context. Additionally, we were dinged in the audit.